connection_util.go

```diff
@@ -153,7 +153,11 @@ func (sc *snowflakeConn) populateSessionParameters(parameters []nameValueParamet
 		sc.cfg.Params[strings.ToLower(param.Name)] = &v
 	}
 	if tz, ok := sc.cfg.Params["timezone"]; ok {
```

Comment:
Could there be multiple connections using different timezones?  And will they update the same localLocation?

Reply:
I would imagine there is only one timezone per connection, and thus this attribute is specific to a single connection instance.

Reply:
what about this 'localLocation'? this var is updated by a connection parameter.